pkg/config/config.go

```diff
@@ -96,6 +96,7 @@ type Config struct {
 	ConfigFolder                 string                `json:"configFolder"`
 	LogLevel                     string                `json:"logLevel"`
 	Logger                       map[string]string     `json:"logger"`
+	RemoteMining                 bool                  `json:"remoteMining"`
```

Comment:
you don't need this anymore. Now we have `RemoteDBHost string` when this is specified it should just do remote mining. When you change it also mention it in the changelog